pycbc/inference/models/gaussian_noise.py

```diff
@@ -562,6 +564,11 @@ def from_config(cls, cp, data_section='data', data=None, psds=None,
             # convert to frequency domain and get psds
             stilde_dict, psds = fd_data_from_strain_dict(opts, strain_dict,
                                                          psd_strain_dict)
+            logging.info('Calculating psds using filter psd')
+            psds = {}
+            for det, d in psd_strain_dict.items():
+                psds[det] = d.filter_psd(opts.psd_segment_length[det],
+                                         strain_dict[det].delta_f, 0.)
```

Comment:
To do: only do this for the gated gaussian.

Reply:
Fixed by creating a class-specific wrapper around `fd_data_from_dict`. For the `GatedGaussian`, this will ensure that `invpsd-trunc-method` (see #3774) is set to `hann` and that the low frequency cutoff for psd estimation is set to 0. Using a hann window for inverse psd truncation turned out to be the difference between `filter_psd` and the psd estimation from the command line tools that was causing a problem. I've removed the call to filter psd as a result, since it was ignoring the psd-estimation argument and always using `median`.